inputfiles/patches/cssom.kdl

```diff
@@ -6,4 +6,13 @@ interface CSSStyleDeclaration \
   forward=CSSStyleDeclarationBase \
   forwardExtends=CSSStyleProperties
 
-interface CSSStyleProperties replaceReference=CSSStyleDeclaration
+interface CSSStyleProperties replaceReference=CSSStyleDeclaration {
+  property imageOrientation deprecated=#true
```

Comment:
We should probably skip this one. I did this for some reason in 0506730e1d1ca9260620189c4f6c8ca19716f0ba without a comment (this is why every patch should ideally have a comment 🥹, but presumably it was because of https://github.com/w3c/csswg-drafts/commit/600fd752153edbe15326a04762334ede1872ad84). But in 2025 all browsers have it and I don't think anyone would unship this.